values.yaml

```diff
@@ -478,7 +478,12 @@ servicePreview:
 
 # Configure the ambassador agent
 agent:
-  enabled: false
+  enabled: true
```

Comment:
👍

Reply:
i had to undo that because of the builds here, which i think is a fine thing to do since edgectl sets the enabled to true. when we get close to merging story/catalog into master we can flip this? https://github.com/datawire/ambassador-chart/pull/170/checks?check_run_id=1954960655